### PR TITLE
feat(splitter): Created functionality for vertical splitter between monitoring and authoring

### PR DESCRIPTION
**Pull request by pavlovicnemanja**

### CODE REVIEW

**Comment by ioanpocol** [comment]
:+1:

**Comment by Aca-jov** [comment]
:+1:

File: client/app/scripts/superdesk/ui/ui.js [hunk 1]
```diff
@@ -1142,8 +1142,83 @@ define([
...
 
+                /*
+                 * If authoring is not initialized,
+                 * wait, and initialize it again
+                 *
+                 * This issue is observed when you are
+                 * switching from settings back to monitoring
+                 */
+                if (!authoring.length) {
+                    $timeout(function () {
```
Comment (petrjasek) [inline client/app/scripts/superdesk/ui/ui.js]: you can set third param to false not to trigger another digest if you only change css

Comment (pavlovicnemanja) [inline client/app/scripts/superdesk/ui/ui.js]: Third param for $timeout?

Comment (petrjasek) [inline client/app/scripts/superdesk/ui/ui.js]: yep third to $timeout. it should be still fine, as long as you don't need any binding to update, only rendering. if you would make some changes to scope then you would need a digest

Comment (pavlovicnemanja) [inline client/app/scripts/superdesk/ui/ui.js]: :+1:

**Pull request merged**